CMakePresets.json

```diff
@@ -184,15 +184,24 @@
     },
     {
       "name": "ci-linux-x64-static-libssl",
-      "inherits": [ "ci-linux-static-libssl", "ci-linux-x64" ]
+      "inherits": [
```

Comment:
these were pretty printed by `jq`, so I conceded to leave them that way because I anticipate continually modifying this JSON file in scripts until the Mbed-TLS vs. OpenSSL flapping is settled.